fs-storage/Cargo.toml

```diff
@@ -9,3 +9,17 @@ crate-type = ["rlib"]
 bench = false
 
 [dependencies]
+data-error = { path = "../data-error" }
+data-json = { path = "../data-json" }
+fs-atomic-versions = { path = "../fs-atomic-versions" }
+data-resource = { path = "../data-resource" }
+log = { version = "0.4.17", features = ["release_max_level_off"] }
+
+serde_json = "1.0.82"
```

Comment:
`serde_json` is only needed for the binary. We shouldn't depend on it for the rest of the package. 

I came across a similar question on stackoverflow. Here is [an answer with some suggestions](https://stackoverflow.com/a/46017284)